dockers/tools/cli/loadtest/Dockerfile

```diff
@@ -0,0 +1,89 @@
+#
+# Copyright (C) 2019-2020 Vdaas.org Vald team ( kpango, rinx, kmrmt )
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+FROM vdaas/vald-base:latest AS builder
+
+ENV ORG vdaas
+ENV REPO vald
+ENV APP_NAME loadtest
+ENV PKG tools/cli/${APP_NAME}
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    libhdf5-dev \
+    && ldconfig \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+
+WORKDIR ${GOPATH}/src/github.com/${ORG}/${REPO}/internal
+COPY internal .
+
+WORKDIR ${GOPATH}/src/github.com/${ORG}/${REPO}/apis/grpc
+COPY apis/grpc .
+
+WORKDIR ${GOPATH}/src/github.com/${ORG}/${REPO}/pkg/${PKG}
+COPY pkg/${PKG} .
+
+WORKDIR ${GOPATH}/src/github.com/${ORG}/${REPO}/cmd/${PKG}
+COPY cmd/${PKG} .
+
+WORKDIR ${GOPATH}/src/github.com/${ORG}/${REPO}
+COPY versions/GO_VERSION .
+COPY versions/VALD_VERSION .
+COPY .git .
+RUN GO_VERSION="$(cat GO_VERSION)" \
+    && VALD_VERSION="$(cat VALD_VERSION)" \
+    && GIT_COMMIT="$(git rev-list -1 HEAD)" \
+    && CPU_INFO_FLAGS="$(cat /proc/cpuinfo | grep flags | cut -d " " -f 2- | head -1)" \
+    && GOOS="$(go env GOOS)" \
+    && GOARCH="$(go env GOARCH)" \
+    && CGO_ENABLED=1 \
```

Comment:
CGO required?

Reply:
yes, we need cgo for hdf5 implementation.